server/models/transactionModel.js

```diff
@@ -0,0 +1,35 @@
+const { DataTypes } = require("sequelize");
+const sequelize = require("../database/sequelize");
+
+const Transaction = sequelize.define(
+  "transaction",
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      allowNull: false,
```

Comment:
remove    `allowNull: false,`

Reply:
removed `allowNull: false` for primary keys in commit 9a6b72d22fc7b1046356ec2c70ed77825b0bb0b4